runtime/memory.x

```diff
@@ -15,7 +15,7 @@ NSTACK_ORG  = 0x5001FC00;
 
 ROM_SIZE    = 32K;
 MBOX_SIZE   = 128K;
-ICCM_SIZE   = 128K;
+ICCM_SIZE   = 112K;
```

Comment:
Why is ICCM smaller? What's taking up the other 14K?

Reply:
I believe this is meant to represent the 16K that was consumed by FMC?

Reply:
The FMC is taking 16k. FMC/RT were overlapping

Reply:
I believe this could be addressed in a separate PR, but as Jordan commented, this is not very intuitive.

A possible suggestion would be formally splitting the ICCM regions for FMC and RT
   _ICCM_FMC_ORG
   ICCM_FMC_SIZE
   ICCM_RT_ORG
   ICCM_RT_SIZE_

Reply:
Expecting VishalM to give us the  full solution. I am adding a workaround so the CI does not fail because of the overlap.

Reply:
For this PR can we add comments in the linker script explaining this?

Reply:
@jhand2  . VishalM has his pr up for review. let's do the partition and comments there - i added you as a reviewer.